docs/antrea-network-policy.md

```diff
@@ -526,6 +526,9 @@ since Pod IPs are ephemeral and unpredictable.
 select Fully Qualified Domain Names (FQDNs), specified either by exact name or wildcard
 expressions, when defining `egress` rules.
 
+**nodeSelector**: This selects certain Node IPs as ingress from address or egress to address.
+It is applicable only to the `from` section in an `ingress` block or the `to` section in an `egress` block.
```

Comment:
Please refer to the existing doc for `podSelector` and `namespaceSelector` fields

Reply:
Updated.